.github/workflows/pr-integration.yml

```diff
@@ -77,29 +81,11 @@ jobs:
       # With a timeout of 300 seconds, try to connect to localhost:8080 every 1 second.
```

Comment:
add shell

Reply:
The `shell` declaration is required in the action and not available in the workflow. I assume they will normalize this at some point.